### ISSUE TITLE
Support Request: The extension does not correctly detect the aspect ratio in the DASH-live streams.

### ISSUE THREAD

**Issue by SlazE888** [Palethorn/native-adaptive-streaming#76]
As you can see in this example[http://trk-dialog.com:8080/dash/camera.mpd], the image is shown compressed horizontally, and looks like in [4x3] format, but in any other player that can receive aspect ratio correctly, you could see that the video is shown in [16x9] format. Now your extension currently plays the 'Source Aspect Ratio' but should reproduce the 'Derived Aspect Ratio'. Please fix it.

**Comment by Palethorn** [comment]
There was a bug which was preventing player resize based on resolution of the browser window. I'm unable to reproduce this issue right now. Are you able to reproduce this issue on version 4.0.0

**Comment by SlazE888** [comment]
> There was a bug which was preventing player resize based on resolution of the browser window. I'm unable to reproduce this issue right now. Are you able to reproduce this issue on version 4.0.0?

Yes I can reproduce this issue on v4.0.0. This is how this TV-channel must looks[In VLC v3.0.11 it have AR equal 16x9]:
![VLC v3 0 11](https://user-images.githubusercontent.com/53570520/102025223-9eeb8500-3da7-11eb-98a0-f66dfd97a7ab.jpg)
and here how it looks in your extension_v4.0.0:
![Untitled](https://user-images.githubusercontent.com/53570520/102025285-ef62e280-3da7-11eb-8b5f-03cb1ab782a9.jpg)
as you can see the image looks flattened horizontally, which is why I asked you to add as workaround my support request #67. Thank you.
